src/components/FeeSelector/index.tsx

```diff
@@ -110,18 +127,45 @@ const FeeSelectorWrapper = styled.div`
   z-index: 3;
 `
 
-const SelectWrapper = styled.div<{ show: boolean }>`
+const SelectWrapperOuter = styled.div<{ show: boolean }>`
```

Comment:
We should use `$show` or React will complain about the invalid HTML prop...

Reply:
dont worry, it shows nothing from my end. I remember that this only happen with `active` props